### PR TITLE
Use complex module definition for edge and api-docs

### PR DESCRIPTION
**Pull request by karelhala**
#### No app loading for edge and api-docs

Currently no app is being loaded in edge or api-docs because of first level navigation. This PR changes the module definition for these 2 apps so it uses complex module definition so we can use all partials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,10 @@ To indicate federated modules scope of your application (you can have multiple s
 
 To indicate which module should be loaded when rendering your app (you can have multiple modules per one scope). This is usually `./RootApp`
 
+##### app_id.frontend.module.group
+
+If you have first level application to indicate which group should be managed by this module.
```
Comment (kruai) [inline README.md]:
```suggestion
If you have a first-level application, this field indicates which group should be managed by this module.
```

File: main.yml [hunk 2]
```diff
@@ -707,9 +714,6 @@ rhel-sw:
 
 ros:
   title: Resource Optimization
-  api:
-    versions:
-      - v0
```
Comment (kruai) [inline main.yml]: Was this change intentional?

**Review by kruai — CHANGES_REQUESTED** [review]

File: main.yml (hunk 2 above)
Comment (karelhala) [reply main.yml]: Good catch!

**Review by kruai — APPROVED** [review]
